lib/Widget/RssProvider.php

```diff
@@ -160,6 +160,7 @@ public function fetchData(DataProviderInterface $dataProvider): WidgetProviderIn
                 $article->link = $item->getUrl();
                 $article->date = Carbon::instance($item->getDate());
                 $article->publishedDate = Carbon::instance($item->getPublishedDate());
+                $article->categories = $item->getCategories();
```

Comment:
What gets returned here? Is it a comma separated list of categories? It is an array?

I am not clear how this gets mapped 1 to 1 with an element?

Reply:
Yeah, it's a comma separated list. Not sure what we wanted to implement for the categories element, but for now it just outputs all the categories with that string. Do you think we should save them separately as an array and use the elements with an index?

That might become confusing since we already have index/slot for each one of the Ticker records... 🤔

Reply:
The question is whether we think a comma separated string is useful?